chat-plugins/mafia.js

```diff
@@ -1244,6 +1249,62 @@ const commands = {
 		},
 		hosthelp: [`/mafia host [user] - Create a game of Mafia with [user] as the host. Requires + % @ * # & ~, voice can only host themselves.`],
 
+		q: 'queue',
+		queue: function (target, room, user) {
+			if (!room.mafiaEnabled) return this.errorReply(`Mafia is disabled for this room.`);
+			if (room.id !== 'mafia') return this.errorReply(`This command can only be used in the Mafia room.`);
+			const args = target.split(',');
```

Comment:
Since your toIding the args alot, why not just do it here?
`const args = target.split(',').map(toId);`